sfepy/discrete/fem/mesh.py

```diff
@@ -386,7 +386,11 @@ def _get_io_data(self, cell_dim_only=None):
         cmesh = self.cmesh
         conns, mat_ids = [], []
         if cell_dim_only is not None:
-            descs = [ii for ii in self.descs if int(ii[0]) == cell_dim_only]
+            try:
+                iter(cell_dim_only)
```

Comment:
Maybe use `is_sequence()` from sfepy.base.base

Reply:
It works but I'm not so fast.